### ISSUE TITLE
[Package Request] Aquantia AQC multigigabit NIC linux driver

### ISSUE THREAD

**Issue by zachberger** [SynoCommunity/spksrc#4499]
* Name: Aquantia AQC multigigabit NIC linux driver 
* Description: driver for many 10GbE NICs
* Website: https://github.com/Aquantia/AQtion
* Software documentation: 
* Build/Installation documentation: https://github.com/Aquantia/AQtion/blob/master/README.txt
* Source code:  https://github.com/Aquantia/AQtion
* License: GPLv2

**Comment by publicarray** [comment]
Licence is **GPLv2** https://github.com/Aquantia/AQtion/search?q=GPL&type=code also see https://github.com/Aquantia/AQtion/issues/7

**Comment by zachberger** [comment]
Thanks, corrected the license in the original post

**Comment by zachberger** [comment]
I think this was accidentally closed, #4526 doesn't actually contain the build for the kernel driver, but rather lays the foundation for it

**Comment by th0ma7** [comment]
@zachberger indeed, the framework for a future package to be created was merged but no PR yet with a formal package build.
